doc/versions.json

```diff
@@ -35,7 +35,7 @@
       "zephyr": "2.6.99",
       "mcuboot": "1.7.99",
       "nrfx": "2.5",
-      "tfm": "1.3.99",
+      "tfm": "1.3.0",
```

Comment:
tfm is later changed to `1.3.99`, why is changed to `1.3.0` here? commit mentions nrfxlib: 00347b8b60f7527311ffcfcdd2994a78842d8cca

Reply:
Fixed.